database/src/main/scala/cromwell/database/SqlDatabase.scala

```diff
@@ -10,184 +9,177 @@ import scalaz.NonEmptyList
 
 
 trait SqlDatabase extends AutoCloseable {
-  protected def isTransient(throwable: Throwable): Boolean = {
-    throwable match {
-      case e: SQLTransientException => true
-      case _ => false
-    }
-  }
-
-  protected def createWorkflow(workflowExecution: WorkflowExecution,
+  def createWorkflow(workflowExecution: WorkflowExecution,
```

Comment:
This (and the other methods in this file) are actually no longer required, I believe